CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/CollectionRequestHandler.swift

```diff
@@ -0,0 +1,142 @@
+//RequestHandler.swift
+//CBLTestServer-iOS
+
+import Foundation
+import CouchbaseLiteSwift
+
+public class CollectionRequestHandler {
+    public static let VOID: String? = nil
```

Comment:
if this is a known nil value, why can't you return directly 'nil' . instead of declaring optional String and then assign nil to it.? is this to avoid some type conflict?